### DISCUSSION TITLE
MacOS???

### DISCUSSION THREAD

**Discussion by mlewis1973** [iacchus/birdears#59] [Birdears Discussion Forum]
is birdears linux specific, or is there a trick to get it working on MacOS?

**Comment by iacchus** [comment]
It isn't necessarily gnu/linux specific; I don't know the situation of SoX (the binary backend we use to play the notes) in MacOs, But the software is written in python, I believe the needed trick would be to install SoX and point to the `play` command binary.

**Reply by mlewis1973** [reply]
thanks.....although quite old, there are macos binaries available for SoX that work.
The other issue is that the latest versions of the click python package are incompatible with birdears (get_terminal_size() was removed with version 8.1.0). pip install click==8.0.4

**Comment by iacchus** [comment]
Thank you, do you did something else to make it run? I want to fix it ASAP, as `click` has this function.

I remembered something else also, I was about to deprecate the `--cli`, but another contributor liked it and changed my mind to keep it,

The following lines take in account GNU/Linux and windows, I don't know if this will work on MacOs:
  
https://github.com/iacchus/birdears/blob/2ef72cca4c9826ae39d3e6189995262016fd5fa9/src/birdears/__init__.py#L248-L287

If you want to try it out, would help people using your O.S..
